packages/runtime/container-runtime/package.json

```diff
@@ -251,15 +244,18 @@
 				"backCompat": false,
 				"forwardCompat": false
 			},
```

Comment:
Are there some things that are effectively permanently broken?

Reply:
There aren't supposed to be. That's not the design intent of the type tests anyway. A permanently broken test would imply that the type is breaking compat every release - which seems obscene for a public API. :) I'm particularly surprised that the ContainerRuntime class declaration is not compatible already. I'm going to start a thread with the runtime experts and see what change they've been making.